code/datums/ruins/lavaland.dm

```diff
@@ -53,6 +53,14 @@
 	cost = 20
 	allow_duplicates = FALSE
 
+/datum/map_template/ruin/lavaland/dwarf
+	name = "Strange Dorf Portal"
+	id = "dorf-man"
+	description = "A portal emitting other-worldly energies, strange pixelated images fade in and out from inside its blocky frame."
+	suffix = "lavaland_surface_dorffort.dmm"
+	cost = 0
```

Comment:
This needs to not have a cost of 0. There is no reason for this to spawn every single round, even while testmerged. It will just run it into the ground

Reply:
This is goof's intention, [he basically told changelingrain otherwise](https://github.com/tgstation/tgstation/pull/29833#issuecomment-322025359) last time he was pulled up on it without compromising on ideas to make the changes to LL (adamantine spawning for instance) less dwarf centric on attitudes of not giving station mining or pre-techwebs RND any minerals or use from it either.

Reply:
Okay, fair enough. But that's stupid, so let's ignore him and change it anyway.